portfolio.html

```diff
@@ -82,5 +82,237 @@ <h2>I'm Amrendra Glad to see you!</h2>
+
+      <div class="acordianwrapper">
+        <button class="accordion">Languages</button>
+        <div class="panel">
+          <div class="languagename">
+            <img src="/Ellipse_JS.png" alt="JSIcon" />
+            JavaScript
+          </div>
+          <div class="languagename">
+            <img src="./Ellipse_HTML.png" alt="HTMLIcon" />
+            HTML
+          </div>
+          <div class="languagename">
+            <img src="./Ellipse_CSS.png" alt="CSSIcon" />
+            CSS
+          </div>
+        </div>
+
+        <button class="accordion">Frameworks</button>
+        <div class="panel">
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat.
+          </p>
+        </div>
+
+        <button class="accordion">Skills</button>
+        <div class="panel">
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat.
+          </p>
+        </div>
+
+        <script>
+          var acc = document.getElementsByClassName("accordion");
+          var i;
+          for (i = 0; i < acc.length; i++) {
+            acc[i].addEventListener("click", function () {
+              this.classList.toggle("active");
+              var panel = this.nextElementSibling;
+              if (panel.style.maxHeight) {
+                panel.style.maxHeight = null;
+              } else {
+                panel.style.maxHeight = panel.scrollHeight + "px";
+              }
+            });
+          }
+        </script>
+      </div>
+    </section>
+    <section class="contactmecard">
+      <div class="contactme">Contact me</div>
+      <div class="contactmecontent">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </div>
+      <form>
+        <input type="text" id="fname" name="fname" value="Amrendra K" /><br />
+        <input
+          type="text"
+          id="lname"
+          name="lname"
+          value="amrendrak.ind@gmail.com"
+        /><br />
+        <!-- <textarea class="textarea" type="text" id="lmessage" name="lmessage" placeholder="Write your message here"><br> -->
+        <textarea
+          class="textarea"
+          name="txtarea"
+          id="txtarea"
+          cols="30"
+          rows="10"
+          placeholder="Write your message here"
+        ></textarea>
+        <button type="button" class="btnseeproject btngetittouch">
+          Get It Touch
+        </button>
+      </form>
+    </section>
```

Comment:
- [ ] [OPTIONAL] In this milestone you do not need to implement the "Contact" section. But, well done, you create a great job! 👍